src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -296,7 +296,7 @@ class FunctionBuilder[F >: Null](val parameterTypeInfo: Array[MaybeGenericTypeIn
 
       cn.accept(cw)
       bytes = cw.toByteArray
-      // CheckClassAdapter.verify(new ClassReader(bytes), false, new PrintWriter(sw1))
+       CheckClassAdapter.verify(new ClassReader(bytes), false, new PrintWriter(sw1))
```

Comment:
Is it intentional to have this enabled in the committed code ?  It looks to me
as though it's for debugging

Reply:
no, I accidentally left it uncommented from when I was debugging :) I'll take it back out.

Reply:
(done)